plugins/web/ui/src/lib/components/IconButton.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
```

Comment:
This is not showing anywhere, whereas it needs to be on blocks' Edit and Delete buttons.